src/mailingui/components/buttons/Buttons.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, ReactNode, CSSProperties } from "react";
+import { Button as ReactEmailButton } from "@react-email/components";
+import { Property } from "csstype";
+
+interface ButtonProps {
+  paddingX?: number;
+  paddingY?: number;
+  href: string;
+  children: ReactNode;
+  backgroundColor?: Property.Color;
+  fontSize?: Property.FontSize<string | number>;
+  color?: Property.Color;
+  borderColor?: Property.Color;
+  pill?: boolean;
+}
```

Comment:
You are already using `React.CSSProperties`, you can extend all of those types as well for your interface (i.e.,`backgroundColor: CSSProperties['color']`) and it should work.

Any reason why using types from `csstype`?